src/integration-tests/src/secret_manager/openapi.rs

```diff
@@ -60,9 +60,7 @@ pub async fn run(config: Option<gax::options::ClientConfig>) -> Result<()> {
                     smo::model::Replication::default()
                         .set_automatic(smo::model::Automatic::default()),
                 )
-                .set_labels(
```

Comment:
Note how *using* these setters gets much easier, here and in other integration tests.